test/models/test_gpt.py

```diff
@@ -127,32 +183,296 @@ def decoder(decoder_layer, num_layers):
 
 
 @pytest.fixture
-def mm_decoder(
-    in_token_emb, out_token_emb, in_pos_emb, out_pos_emb, decoder, right_shift
+def mm_decoder(in_pos_emb, out_pos_emb, decoder, right_shift):
+    return MultimodalTransformerDecoder(in_pos_emb, out_pos_emb, decoder, right_shift)
+
+
+@pytest.fixture
+def tokenizer(num_emb, emb_dim):
+    class DummyTokenizer(nn.Module):
```

Comment:
Not sure if this is the right place for it or not, but it might be worthwhile to test that things work with a nontrivial tokenizer somewhere as well.